core/src/lib/auth/providers/userapikey/CoreUserAPIKeyAuthProviderClient.ts

```diff
@@ -19,150 +19,149 @@ enum ApiKeyFields {
 /**
  * A client for the user API key authentication provider which can be used to obtain a credential for logging in.
  */
-export default abstract class CoreUserAPIKeyAuthProviderClient extends CoreAuthProviderClient<StitchAuthRequestClient>{
+export default abstract class CoreUserAPIKeyAuthProviderClient extends CoreAuthProviderClient<
+  StitchAuthRequestClient
```

Comment:
[nit] this looks really strange. was this the prettier? It does not look pretty haha. can you just line break before the `extends` and keep the generic information all on one line